src/evaluator.rs

```diff
@@ -240,7 +240,8 @@ impl<'a> Evaluator<'a> {
                 | component::Type::S8
                 | component::Type::S16
                 | component::Type::S32
-                | component::Type::S64 => self.lookup_in_scope(ident),
+                | component::Type::S64
+                | component::Type::String => self.lookup_in_scope(ident),
```

Comment:
I'm trying to remember why we wouldn't just always look up in scope if it's not one of the special types we'd like to handle... I wonder if we should instead get rid of this arm and turn it into:

```rust
_ => self.lookup_in_scope(ident)
```

Reply:
I'm happy to provide an additional PR to head into that direction. Unfortunately, I saw your comment after you merged the PR